### ISSUE TITLE
How can I keep the menu right opening all time?

### ISSUE THREAD

**Issue by tanhoang181** [aryaxt/iOS-Slide-Menu#167]
Thank you for very useful lib. But I have a little issue relate to UX in my app.

I use SlideNavigationController with a Home view controller (root) and a Menu view controller (right menu). I toggle menu right to open Menu VC. I press a button in the Menu VC, I present an UINavigationController. Then I dismiss it. And I see the menu right was closed.

How can I prevent the menu right from closing?

Thanks.

**Comment by gabrielribeiro** [comment]
Got any idea?

**Comment by tanhoang181** [comment]
No. This lib has no way to do that. The other lib "PKRevealController" works :)

**Comment by gabrielribeiro** [comment]
@TheTonon
